cdap-api/src/main/java/co/cask/cdap/api/data/batch/Input.java

```diff
@@ -218,10 +224,22 @@ private DatasetInput(String name, Map<String, String> arguments, @Nullable Itera
   public static class StreamInput extends Input {
 
     private final StreamBatchReadable streamBatchReadable;
+    private String namespace;
 
     private StreamInput(StreamBatchReadable streamBatchReadable) {
       super(streamBatchReadable.getStreamName());
       this.streamBatchReadable = streamBatchReadable;
+      this.namespace = null;
```

Comment:
We don't have this null initialization explicit in DatasetInput. We don't need it, right?